src/component/common/Dialogue/Dialogue.tsx

```diff
@@ -68,18 +70,24 @@ const Dialogue: React.FC<IDialogue> = ({
                 <ConditionallyRender
                     condition={Boolean(onClick)}
                     show={
-                        <Button
-                            form={formId}
-                            color="primary"
-                            variant="contained"
-                            onClick={handleClick}
-                            autoFocus={!formId}
-                            disabled={disabledPrimaryButton}
-                            data-test={DIALOGUE_CONFIRM_ID}
-                            type={formId ? 'submit' : 'button'}
-                        >
-                            {primaryButtonText || "Yes, I'm sure"}
-                        </Button>
+                        <ConditionallyRender
+                            condition={Boolean(permissionButton)}
+                            show={permissionButton}
+                            elseShow={
```

Comment:
This button previously only showed if there was an onClick handler passed. With the current code it will always show, even if an onClick handler is not passed to the dialogue.